src/test/java/dk/kb/datahandler/pvica/PvicaDataTest.java

```diff
@@ -21,7 +22,13 @@ public void testDeliverableUnitNameSpaceFix() throws Exception{
     }
 
 
-    @Test
+    public void testFindPvicaParent() throws Exception{
+        String xmlFile = "xml/pvica_parent_test.xml";        
+        String xml = Resolver.resolveUTF8String(xmlFile);        
+        String parent = OaiResponseFiltering.getPvicaParent(xml);
+        assertEquals("6a8d279b-0276-4856-b2ba-77b0162f5d63", parent);        
+    }
+
     public void testManifestationNameSpaceFix() throws Exception{
```

Comment:
This one has also lost its Test statement in the merge, can we have it re-added?